src/phoenix/experiments/functions.py

```diff
@@ -95,6 +95,7 @@ def run_experiment(
     dry_run: Union[bool, int] = False,
     print_summary: bool = True,
     concurrency: int = 3,
+    timeout: Optional[int] = None,
```

Comment:
Could you add this Args to the documentation on line 151?

Reply:
thanks for the quick response. it's done.